### ISSUE TITLE
Default plotting range not working for uniform distributions

### ISSUE THREAD

**Issue by niklhart** [JonasMoss/univariateML#49]
Function `plot_wrangler` (and hence the `plot`, `lines` and `points` methods) doesn't work for uniform distributions if `range = NULL`:

```
library(univariateML)
plot(mlunif(0:1))
```

produces the following error:
```
Error in abs(support[1]) : non-numeric argument to mathematical function
```

**Comment by JonasMoss** [comment]
Cheers. Fixed in 92b79423233e21c1cc716145fcceb197b0a92f0c. Will submit to CRAN once the CI is updated.

**Comment by niklhart** [comment]
Great, thanks a lot!
